### PR TITLE
EVAKA-4259 personal details finishing touches

### PR DESCRIPTION
**Pull request by jaahon**
#### Summary

- Add personal details page's Swedish and English translations
- Remove personalDetailsPage feature flag

### CODE REVIEW

**Review by kimifoud — COMMENTED** [review]
Everything looks good! Ship it in once @zch approves `sv`.

**Comment by akheron** [comment]
Translations came from a translation agency, so they should be fine, right? @zch just added a few missing ones.

**Review by kimifoud — APPROVED** [review]

**Pull request merged**
